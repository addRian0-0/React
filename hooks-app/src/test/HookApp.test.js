import { shallow, configure } from 'enzyme';
import { HookApp } from "../HookApp";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

configure({ adapter: new Adapter() });

describe('Probar el entorno de configuracion de setupTest', () => {

    const wrapper = shallow(<HookApp />)

    test('hacer snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    })

})