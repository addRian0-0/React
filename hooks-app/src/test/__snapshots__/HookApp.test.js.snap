// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probar el entorno de configuracion de setupTest hacer snapshot 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
